doc/classes/HeightMapShape.xml

```diff
@@ -4,7 +4,7 @@
 		Height map shape for 3D physics (bullet only)
 	</brief_description>
 	<description>
-		Height map shape resource, which can be added to a [PhysicsBody] or [Area].
+		Height map shape resource, which can be added to a [PhysicsBody] or [Area]. Note that bullet will always center the collision shape. If you minimum height is 0, and you maximum height is 10, bullet will adjust your collision shape down so the minimum height is -5 and the maximum height is 5. 
```

Comment:
I know heightmap isn't supported by Godot Physics, but perhaps we could indicate this is a Bullet-only behavior?

Reply:
It's in the brief description:
Height map shape for 3D physics **(bullet only)**

Reply:
Could you capitalize "Bullet", to make it clear it's the library and not a bullet object/scene?

Reply:
Also, trailing space on this line :)

Reply:
All good :)